File: topics/arrays/exercises/Contains.pas
function ????(const data: array of Integer; val: Integer): Boolean;
var
    i: Integer;
begin
    result := False;
    
    for i := Low(data) to High(data) do
    begin
        if data[i] = val then
        begin
            result := True;
            exit;
        end;
    end;
end;

File: topics/arrays/exercises/contains.c
bool ????(const int data[], int sz, int val)
{
    int i;
    bool result = false;
    
    for (i = 0; i < sz; i++)
    {
        if (data[i] == val)
        {
            result = true;
            break;
        }
    }
    
    return result;
}